scripts/singlecell/geneformer/test_train.py

```diff
@@ -29,7 +29,7 @@
 from bionemo.testing.data.load import load
 
 
-data_path: Path = load("single_cell/testdata-20240506") / "cellxgene_2023-12-15_small" / "processed_data"
+data_path: Path = load("single_cell/testdata-memmap-format") / "cellxgene_2023-12-15_small_mmap"
```

Comment:
Could you actually put this under a pytest fixture?

```suggestion
@pytest.fixture
def data_path() -> Path:
    return load("single_cell/testdata-memmap-format") / "cellxgene_2023-12-15_small_mmap"
```

You'd then also need to add `data_path` as an argument to functions that use it